### DISCUSSION TITLE
Searching on tab

### DISCUSSION THREAD

**Discussion by sollermun** [kimai/kimai#3308] [General]
When creating a new time entry, it would be nice if I could tab to customer, project, or activity and just start typing to search. Right now, I have to manually click the field with the mouse before I can begin typing to search. It's a small annoyance, but an annoyance nonetheless.

I'm using Chrome on a PC. I'm not sure if this behavior exists on other browsers.

**Comment by kevinpapst** [comment]
You have to hit space or enter to start typing.
The next major release will include a new library for the dropdowns, which has a better (more native) behavior.
